src/ventus/wm/windowmanager.js

```diff
@@ -102,9 +108,21 @@ function($, Window, View, DefaultMode, ExposeMode, FullscreenMode) {
 		get overlay() {
 			return this._overlay;
 		},
+		
+		set margin(marginObj) {
+			if (typeof marginObj !== 'object') {
+				return;
+			}
+			this._margin = marginObj;
+			return this;
+		},
+		
+		get margin() {
+			return this._margin;
+		},
 
 		createWindow: function(options) {
-			var win = new Window(options);
+			var win = new Window(this, options);
```

Comment:
I'm a little bit worried about the new dependency between Window and WindowManager, do you think this will work if we send just the margin object?

Or maybe a getter, in case `_margin` reference is modified:

    options.getMargins = function() {
      return this._margin;
    }.bind(this);
    var win = new Window(options);